platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -370,8 +362,6 @@ public class SwirldsPlatform implements Platform {
 
         registerAddressBookMetrics(metrics, currentAddressBook, selfId);
 
```

Comment:
Are you intentionally not starting the `recycleBin` anymore?

Reply:
Oops, that's an oversight. We were starting it via introspection, not by contract. I've updated the interface and added it to things to start.